http/src/v0/refs.rs

```diff
@@ -238,17 +244,29 @@ pub enum Loaded {
     Ipld(Ipld),
 }
 
+#[derive(Default, Debug)]
```

Comment:
if we're expecting to usually be following `dagpb` data it might make sense to introduce a manual `Default` impl (with `follow_dagpb_data: true`), which would allow us to create the opts in a less verbose manner

Reply:
I think this used to be `true` for more places, but according to tests it's needed only in one occasion (refs). Should had written better tests, but I expect to wrestle with this in the next step.